core/amber/src/main/scala/edu/uci/ics/texera/web/ObserverManager.scala

```diff
@@ -0,0 +1,21 @@
+package edu.uci.ics.texera.web
+
+import rx.lang.scala.Observer
+
+import scala.collection.mutable
+
+class ObserverManager[U] {
```

Comment:
this doesn't look right, this is essentially just a multi-cast subject and we shouldn't re-invenet new things. Here's the corresponding functions
`addObserver` -> `subscribe`
`removeObserver` -> `unsubscribe`
`pushToObservers` -> `next`

consider something like 
https://www.javadoc.io/doc/io.reactivex/rxscala_2.10/0.22.0/rx/lang/scala/subjects/PublishSubject$.html

Reply:
Changed it to `WebsocketOutput` and renamed the APIs.